lib/tags.js

```diff
@@ -0,0 +1,14 @@
+module.exports.generateTagsSpec = tags => {
+  return tags.map(t => {
+    return {
+      name: t,
+      description: `${t} resource`
```

Comment:
Not sure if we need the `description` if it's always the same as the name - is it useful in terms of what swagger/redoc renders, or not really?

Reply:
> About the commits - everything's great, the only issue I see is that you've merged the `mpashkovskiy/master` branch into yours. These two commits thus are unnecessary - I'd highly recommend `git rebase`, which once you get used to - the workflow becomes way better + you get a cleaner commit log:D
> 
> The workflow - it's great if I've made changes and I want to keep them up to date with the upstream, or just the branch I want to merge the changes back into:
> 
> ```shell
> # starting from scratch - creating a branch that you'll create a pull request from
> git checkout master
> git pull
> git checkout -b feat/create-an-awesome-feature
> # make changes & commit
> # fetch to get latest changes
> git fetch
> # interactive rebase on top of origin/master (the remote repo & branch you want to merge back into)
> # (in our case you'd have to set a url for the upsteam repo like so:
> git remote add upstream https://github.com/mpashkovskiy/express-oas-generator
> # and use upstream/master instead
> #
> # back to git rebase - it's awesome and you can do cool stuff with it.
> #  For now, execute this command, change nothing and close the file to confirm
> git rebase -i origin/master
> # if there've been conflicts - you'll have to resolve them commit-by-commit. Otherwise you're ready to go
> # now, you'll either have new commits from origin/master, thus you can push the updated changes, using --force to override the old ones
> git push -f
> # NOTE - only do this on branches you're the only one working on or you'll likely mess things up
> ```
> 
> I hope this isn't too intimidating:D One thing's for sure - rebase is pretty powerful and you might want to play around with it on a test project _before_ risking losing your changes/messing something up for a real project:D So I thought that this would be informative and useful, but you definitely do not need to do this for this PR, esp. if you haven't done it before!

The good old `merge vs rebase` discussion 😄 . Will do it the next time!

Reply:
Yes, it renders the **name** and description. Take a look at the example:
![image](https://user-images.githubusercontent.com/5941817/89126960-4da9bb80-d4c0-11ea-879e-85c46e2aec81.png)

Just removed the description, as it doesn't provide much more information 😄. Looks like this now:

![image](https://user-images.githubusercontent.com/5941817/89127003-93668400-d4c0-11ea-8e12-aac750ae8a82.png)

Reply:
ah great, yep this is awesome